TerminalDocs/customize-settings/profile-settings.md

```diff
@@ -0,0 +1,423 @@
+
+**Property name:** `name`
+
+**Necessity:** Required
+
+**Accepts:** String
+
+### Icon
+
+Set the icon that displays within the tab and the dropdown menu.
+
+**Property name:** `icon`
+
+**Necessity:** Optional
+
+**Accepts:** File location as a string
+
+### Hide a profile from the dropdown
+
+If `"hidden"` is set to `true`, the profile will not appear in the list of profiles. This can be used to hide default profiles and dynamically generated profiles, while leaving them in your settings file.
+
+**Property name:** `hidden`
+
+**Necessity:** Optional
+
+**Accepts:** `true`, `false`
+
+**Default value:** `false`
+
+<br />
+
+___
+
+## Tab title settings
+
+### Custom tab title
+
+If set, this will replace the `name` as the title to pass to the shell on startup. Some shells (like `bash`) may choose to ignore this initial value, while others (`cmd`, `powershell`) may use this value over the lifetime of the application. If you'd like to learn how to have the shell set your title, visit the [tab title tutorial](./../tutorials/tab-title.md).
+
+**Property name:** `tabTitle`
+
+**Necessity:** Optional
+
+**Accepts:** String
+
+### Suppress title changes from the shell
+
+When set to `true`, `tabTitle` overrides the default title of the tab and any title change messages from the application will be suppressed. If `tabTitle` isn't set, `name` will be used instead. When set to `false`, `tabTitle` behaves as normal.
+
+**Property name:** `suppressApplicationTitle`
+
+**Necessity:** Optional
+
+**Accepts:** `true`, `false`
+
+<br />
+
+___
+
+## Text settings
+
+### Font face
+
+Name of the font face used in the profile. The Terminal will try to fallback to Consolas if this can't be found or is invalid. For other variants of the default font, Cascadia Code, visit the [Cascadia Code GitHub repository](https://github.com/microsoft/cascadia-code).
+
+**Property name:** `fontFace`
+
+**Necessity:** Optional
+
+**Accepts:** Font name as a string
+
+**Default value:** `"Cascadia Code"`
+
+### Font size
+
+Sets the profile's font size.
+
+**Property name:** `fontSize`
+
+**Necessity:** Optional
+
+**Accepts:** Integer
+
+**Default value:** `12`
+
+### Padding
+
+Set the padding around the text within the window. This will accept three different formats: `"#"` sets the same padding for all sides, `"#, #"` sets the same padding for left-right and top-bottom, and `"#, #, #, #"` sets the padding individually for left, top, right, and bottom.
+
+**Property name:** `padding`
+
+**Necessity:** Optional
+
+**Accepts:** Values as a string in the following formats: `"#"`, `"#, #"`, `"#, #, #, #"`
+
+**Default value:** `"8, 8, 8, 8"`
+
+### Antialiasing text
+
+You can control how text is antialiased in the renderer. Note that changing this setting will require starting a new terminal instance.
```

Comment:
Link out to definition of antialiased? And definitely need some sort of magnified GIF of what they all look like on the edges of the text.